docs/sources/panels-visualizations/visualizations/bar-chart/index.md

```diff
@@ -208,7 +208,7 @@ By setting the width of the axis, data whose axes types are different can share
 
 ### Soft min and soft max
 
-Set a **Soft min** or **soft max** option for better control of Y-axis limits. By default, Grafana sets the range for the Y-axis automatically based on the dataset.
+Set a **Soft min** or **soft max** option for better control of all axis limits. By default, Grafana sets the range for the Y-axis automatically based on the dataset.
```

Comment:
this does not apply to BarChart

Reply:
Fixed in latest commit, accidentally updated the wrong one, apologies!